src/rust-crypto/backup.ts

```diff
@@ -205,22 +205,41 @@ export class RustBackupManager extends TypedEventEmitter<RustBackupCryptoEvents,
      */
     public async importBackedUpRoomKeys(keys: IMegolmSessionData[], opts?: ImportRoomKeysOpts): Promise<void> {
         const keysByRoom: Map<RustSdkCryptoJs.RoomId, Map<string, IMegolmSessionData>> = new Map();
+        let invalid = 0;
         for (const key of keys) {
+            if (typeof key.room_id != "string" || typeof key.session_id != "string") {
```

Comment:
hrm. Did you observe real-world backup files that were malformed in this way?

Reply:
Oh, I just took a look at this again, and the code that calls this sets `room_id` and `session_id`, rather than it coming from unauthenticated data, so we don't need this check after all.  Will remove.